opensfm/test/test_pairs_selection.py

```diff
@@ -117,27 +143,31 @@ def create_match_candidates_config(**kwargs):
     return config
 
 
-def test_match_candidates_from_metadata_vlad(tmpdir):
+def test_match_candidates_from_metadata_vlad(lund_path):
     config = create_match_candidates_config(matching_vlad_neighbors=NEIGHBORS)
```

Comment:
Should we set `matching_gps_distance: 0` to make sure we don't use gps based neighbors in addition to vlad neighbors?